src/helpers/token.ts

```diff
@@ -108,3 +111,21 @@ export function unwrapSOLInstruction(
   }
   return null;
 }
+
+export function setTokenAccountOwnerTx(
```

Comment:
for common use, should be change the name of these params like:
positionNftAccount -> tokenAccount
currentOwnerPubkey -> from
feeVault -> to